pkgs/by-name/pi/pista/package.nix

```
@@ -0,0 +1,27 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "pista";
+  version = "unstable-2023-02-20";
+
+  src = fetchFromGitHub {
+    owner = "oppiliappan";
+    repo = "pista";
+    rev = "378535f3e7e138110b6225616e0dc61066a5605a";
+    sha256 = "sha256-FlWDKz5B/sC+VCtJNmtCJTkxzeOJOMT9gZlG6UVGzKU=";
```

```suggestion
    hash = "sha256-FlWDKz5B/sC+VCtJNmtCJTkxzeOJOMT9gZlG6UVGzKU=";
```

Sorry, I missed this before, it's preferable to use `hash`.